src/graphql/graduateExplorer/service.js

```
@@ -2,6 +2,22 @@ const _ = require('lodash')
 const { eCarmen } = require('../../lib/clients')
 const { getEcarmenPaginationObj } = require('../../lib/data/utils')
 
+const getGraduateExplorerGeoData = async ({ args, data, context }) => {
+  try {
+    const universityDimName = _.get(data, 'education.school.name.dim', '')
```

If there's no `universityDimName` do we want to throw an error? The results probably won't make sense and we probably don't want to expose any data that would come back to anybody.

---

Fixed it. Thanks